src/FSharp.Data.GraphQL.Server/Execution.fs

```diff
@@ -253,6 +253,7 @@ let private resolveField (execute: ExecuteField) (ctx: ResolveFieldContext) (par
 /// Lifts an object to an option unless it is already an option
 let private toOption x = 
```

Comment:
This is a change suggestion made by the issue [158](https://github.com/fsprojects/FSharp.Data.GraphQL/issues/158).